### PR TITLE
change figs to match tree and dataframe

### LINKED ISSUE #50: Docs: update images
- [ ]  fix fig 1 in user guide
- [ ]  user guide call tree and performance data to match

### PR DESCRIPTION
**Pull request by vanessalama09**
The goal of this PR is to replace the figures of the performance data table and the hatchet call tree from the user guide with figures that correspond to the same thicket. 

Fixes #50

### CODE REVIEW

**Comment by slabasan** [comment]
@vanessalama09 Can you also fix fig 1 in the user guide as part of this PR? See https://github.com/LLNL/thicket/issues/50

**Comment by vanessalama09** [comment]
> @vanessalama09 Can you also fix fig 1 in the user guide as part of this PR? See #50

Fixed

File: docs/images/Table-Tree-Revised-gradien.png
Comment (slabasan) [inline docs/images/Table-Tree-Revised-gradien.png:1]: Can we restore the titles with the call tree, multi-prof. metrics, etc?

Comment (vanessalama09) [reply docs/images/Table-Tree-Revised-gradien.png:1]: Fixed!

**Review by slabasan — APPROVED** [review]

**Pull request merged**